app/models/sub_section.rb

```diff
@@ -0,0 +1,72 @@
+class SubSection
+  attr_reader :sub_section_slug
+
+  delegate :title, to: :tag_from_content_api
+
+  def initialize(sub_section_slug)
+    @sub_section_slug = sub_section_slug
+  end
+
+  def lists
+    if curated_links?
+      @lists ||= curated_lists
+    else
+      @lists ||= fallback_list_of_uncurated_links
+    end
+  end
+
+  def curated_links?
+    subsection_from_content_store.details && subsection_from_content_store.details.groups
```

Comment:
This will return true if the subsection contains an empty list of groups.  We need it to return false in that case.